includes/class-freemius.php

```diff
@@ -22006,6 +22006,9 @@ function _fetch_latest_version(
 
             if ( true === $fetch_readme ) {
                 $latest_version_endpoint = add_query_arg( 'readme', 'true', $latest_version_endpoint );
+
+                // Don't cache the API response when fetching readme information.
+                $expiration = null;

```

Comment:
How come this code is here @DanieleAlessandra ? Please have a look. You are targeting `master` instead of `develop` ;)